### PR TITLE
Update instant-booted-simulators.md

### PR DESCRIPTION
**Pull request by eyaly**
Since this is a code example, Yell (A customer) copied/pasted this line of code to their framework as it is and got an error to creating the driver

### Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### CODE REVIEW

**Review by lysannep — APPROVED** [review]
@eyaly Thank you!

**Review by wswebcreation — CHANGES_REQUESTED** [review]
Can you change all values, you now only changed one for JAVA, but we have all languages in here

File: docs/mobile-apps/features/instant-booted-simulators.md [hunk 1]
```diff
@@ -320,7 +328,8 @@ AppiumOptions capabilities = new AppiumOptions();
 
 capabilities.AddAdditionalCapability("browserName", "safari");
 capabilities.AddAdditionalCapability("platformName", "ios");
-capabilities.AddAdditionalCapability("appium:platformVersion", "latest|current_major|previous_major");
+// Possible values: "latest", "current_major", "previous_major"
```
[the remark this answers is not in the document]
Comment (eyaly) [reply docs/mobile-apps/features/instant-booted-simulators.md:331]: Thanks @wswebcreation. I didn't even make the change for Java iPad... So now I added the change for all

**Review by wswebcreation — CHANGES_REQUESTED** [review]
Small improvements, you can accept them and then we can merge it.

Thanks

**Review by wswebcreation — APPROVED** [review]

**Pull request merged**